packages/gensx-storage/README.md

```diff
@@ -245,3 +227,29 @@ const results = await namespace.query({
   topK: 3,
 });
 ```
+
+## Troubleshooting
+
+If you have `@libsql/client` installed as a dependency, it may cause issues for bundlers. You should always install it as a dev dependency as local databases can only be used during development.
```

Comment:
> can only be used during development

Is that actually true? Aren't there cases where someone might want to use a filesystem database (eg production, running on their own server or something)?